scripts/docker-entrypoint.sh

```diff
@@ -418,9 +418,9 @@ if [ -z "${IS_PEGA_25_OR_LATER}" ]; then
     echo "No krb5.conf was specified in ${krb5_conf}."
   fi
 
-  echo "java -cp "\"${vc_classpath}\"" ${vc_javaopts} com.pega.cmc.VersionChecker"
+  echo "java -cp ${vc_classpath} ${vc_javaopts} com.pega.cmc.VersionChecker"
 
-  pega_version=$(eval exec java -cp "\"${vc_classpath}\"" "${vc_javaopts}" com.pega.cmc.VersionChecker)
+  pega_version=$(java -cp ${vc_classpath} ${vc_javaopts} com.pega.cmc.VersionChecker)
```

Comment:
one set of quotes around vc_classpath probably makes sense to prevent it from globbing

Reply:
I found that I had to include `eval exec` or else it choked on some of the java options.